packages/widgets/src/commandpalette.ts

```diff
@@ -1050,6 +1050,11 @@ namespace Private {
    */
   interface IScore {
     /**
+     * The numerical type for the text match.
+     */
+    matchType: number;
+
```

Comment:
typescript has an `enum` type which can be used to better represent numerical constants like this. Also, since it's not exported outside of the module, you can safely make it a `const enum`, which will cause the compiler to inline the numerical values at the call site. So, something like:

```
const enum MatchType {
  None = -1,
  LabelExact = 0,
  LabelFuzzy = 1,
  CategoryExact = 2,
  CategoryFuzzy = 3,
  Mixed = 4
}

interface IScore {
  matchType: MatchType;
  ...
}
```